src/sections/quiz/quiz-header.js

```diff
@@ -0,0 +1,36 @@
+import Button from "@mui/material/Button";
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import Logo from "../../components/svg-color/Logo.svg";
+import {Lora} from '@next/font/google';
+
+const lora = Lora({preload: false});
+
+export default function QuizHeader() {
+  return (
+    <Box
+      component="header"
+      sx={{
+        backgroundColor: '#f9f4ee',
+        padding: '24px',
```

Comment:
Use responsive values, i.e. padding: {3}. That is a value relative to a current font size. This way spacing will scale together with font changes, set in theme typography.

Reply:
I have a syntax error when padding: {3},
changed to 1.8em
![image](https://github.com/digital-brother/wct-fe/assets/49399157/852b3011-31b1-48ab-8c5e-259806dcd5c9)